### PR TITLE
Update command descriptions from old 'property/cluster/namespace' format to current 'tenant/namespace' format

### PR DESCRIPTION
**Pull request by eolivelli**
### Motivation

Some command use the old format "property/cluster/namespace", it is better that the help reflects current version style.

### Modifications

Fix the examples

### Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNonPersistentTopics.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ private NonPersistentTopics getNonPersistentTopics() {
 
     @Parameters(commandDescription = "Lookup a topic from the current serving broker")
     private class Lookup extends CliCommand {
-        @Parameter(description = "non-persistent://property/cluster/namespace/topic", required = true)
+        @Parameter(description = "non-persistent://tenant/namespaces/topic", required = true)
```
Comment (merlimat) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNonPersistentTopics.java:56]: In all the repetitions, it should be:
```suggestion
        @Parameter(description = "non-persistent://tenant/namespace/topic", required = true)
```

**Review by eolivelli — COMMENTED** [review]
@merlimat you are right.
thanks for the comment.

patch updated

File: bouncy-castle/bc/pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.pulsar</groupId>
     <artifactId>bouncy-castle-parent</artifactId>
-    <version>2.8.0-SNAPSHOT</version>
+    <version>2.7.2.1.0.0</version>
```
Comment (merlimat) [inline bouncy-castle/bc/pom.xml:28]: Ok, now there was some merge issue :)

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNonPersistentTopics.java [hunk 3]
```diff
@@ -122,7 +122,7 @@ void run() throws Exception {
 
     @Parameters(commandDescription = "Get list of non-persistent topics present under a namespace")
     private class GetList extends CliCommand {
-        @Parameter(description = "property/cluster/namespace", required = true)
+        @Parameter(description = "tenant/namespaces", required = true)
```
Comment (merlimat) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNonPersistentTopics.java:125]: Where there was the "namespace" only parts is still written in plural form

**Comment by eolivelli** [comment]
@merlimat precommit check passed

**Review by merlimat — APPROVED** [review]

**Pull request merged**